fhir-odoo-mapper/src/main/java/com/ozonehis/fhir/odoo/model/Product.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright © 2024, Ozone HIS <info@ozone-his.com>
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.ozonehis.fhir.odoo.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.io.Serial;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class Product implements OdooResource {
```

Comment:
We should add the mapper logic in `eip-odoo-openmrs`. 
For that we will have to create a sub-module of mapper classes present in `eip-odoo-openmrs` repo and then use it as a dependency in this project.